src/client/lib/_mapbox/wms-layer.js

```diff
@@ -0,0 +1,18 @@
+const url = 'http://tl-ng044.xtr.deltares.nl/geoserver/percelen/wms?SERVICE=WMS&VERSION=1.1.1&REQUEST=GetMap&FORMAT=image%2Fpng&TRANSPARENT=true&STYLES&LAYERS=percelen%3Abrp_gewaspercelen_2017_concept&exceptions=application%2Fvnd.ogc.se_inimage&SRS=EPSG:3857&WIDTH=256&HEIGHT=256&bbox={bbox-epsg-3857}'
```

Comment:
should any of this go to the `.env` file?

Reply:
this will be refactored when we start with the different layers in sprint 1